app/Course/course.html

```diff
@@ -2,7 +2,7 @@
     <a href="#/courseDetail" class="btn btn-default">
         <div class="menu-item-container">
             <div class="course-item-img">
-                <img src="app/css/img/food1.jpg" />
+                <img src="/app/css/img/food1.jpg" />
```

Comment:
@sarah-hoter I think it should be without /app/ at all
I did it such: css/img/food1.jpg and it worked
please change it and try